inventory/sample.byo.example.com.d/inventory/group_vars/OSEv3.yml

```diff
@@ -0,0 +1,50 @@
+---
+
+deployment_type: openshift-enterprise
+openshift_deployment_type: openshift-enterprise
+openshift_master_cluster_method: native
+openshift_release: v3.7
+
+# openshift_master_cluster_public_hostname must be something like:
+#   <prefix>.{{ env_id }}.{{ dns_domain }}
+openshift_master_cluster_public_hostname: master.{{ env_id }}.{{ dns_domain }}
+
+# openshift_master_cluster_hostname must be something like:
+#   <prefix>.{{ env_id }}.{{ dns_domain }}
+openshift_master_cluster_hostname: master-internal.{{ env_id }}.{{ dns_domain }}
+
+# openshift_master_default_subdomain must be something like:
+#   <prefix>.{{ env_id }}.{{ dns_domain }}
+openshift_master_default_subdomain: apps.{{ env_id }}.{{ dns_domain }}
+
+
+# HTPASSWD Identity Provider
+openshift_master_identity_providers:
+ - 'name': 'htpasswd_auth'
+   'login': 'true'
+   'challenge': 'true'
+   'kind': 'HTPasswdPasswordIdentityProvider'
+   'filename': '/etc/origin/master/htpasswd'
+
+#this will create an admin/admin user
+openshift_master_htpasswd_users:
+  admin: $apr1$7aiANAYb$TOUYVUqnBqBlD5AQEIMYw1
+
+openshift_hosted_router_selector: 'region=infra'
```

Comment:
These aren't really needed to be set - any particular reason to include them? (if so, we should include them for the other sample inventories as well). Same for the registry below.